### DISCUSSION TITLE
Mercado Sul-americano

### DISCUSSION THREAD

**Discussion by kscastro** [frontendbr/forum#835] [Discussão]
Fala Pessoal,

Eu tenho tenho curiosidade sobre o mercado sul-americano se existem vagas, pagam bem quais empresas que tem filiais espalhadas por essa região o que esse mercado precisa?

**Comment by rubenmarcus** [comment]
Eu sei que o Equador paga em dolar ahahha

**Comment by rubenmarcus** [comment]
o Jooble e o linkedin são bons locais pra ver as vagas, eu sei que no Equador se paga um pouco menos do que aqui (mas em dolar) e o custo de vida é muito mais baixo do que aqui.

Sei também que Equador e Paraguai o processo de obtenção de cidadania é muito simples, no Paraguai acho que é só um depósito bancario. Mas acho que a obtenção de visto de trabalho varia de país pra país. O jeito seria você pesquisar o país que mais gosta, tirar uma férias de pelo menos 1 mes lá e ja ir mandando curriculo aqui do Brasil. Se uma empresa se interessar, pode ser que banque seu visto. Eu conheci uma redatora do Brasil, que hoje é redatora na Índia e conseguiu o trabalho assim!

aí uns links da America Latina no Jooble (os caras tem no mundo todo)
https://uy.jooble.org/trabajo-front-end
https://mx.jooble.org/trabajo-front+end

Aqui um Programador Junior ganha até 700 dolares mês , no Equador
https://ec.jooble.org/trabajo-front+end

https://cr.jooble.org/trabajo-front+end


Senior na Colombia tira uns 7300 reais 
https://co.jooble.org/trabajo-front+end

Argentina é um mercado bom tambem
https://ar.jooble.org/trabajo-front-end

**Comment by rubenmarcus** [comment]
sobre salário acho que o melhor custo-benefício pra um desenvolvedor é ganhar numa moeda forte e morar num país com custo de vida baixo, ex: Sudeste Asiático (não é a toa que lá é o hub de freelancers no mundo, Tailândia, Indonésia, Vietnã) , Cingapura é muito caro , e Malásia é apenas caro.

**Comment by kscastro** [comment]
Obrigado pelas dicas @rubenmarcus  tenho olhado mais para o mercado sul americano
